ssvsigner/client.go

```diff
@@ -166,20 +165,19 @@ func (c *Client) RemoveValidators(ctx context.Context, pubKeys ...phase0.BLSPubK
 		ToJSON(&resp).
 		Fetch(ctx)
 	if err != nil {
-		return fmt.Errorf("request failed: %w", err)
+		return nil, fmt.Errorf("request failed: %w", err)
 	}
 
 	if len(resp.Data) != len(pubKeys) {
-		return fmt.Errorf("unexpected statuses length, got %d, expected %d", len(resp.Data), len(pubKeys))
+		return nil, fmt.Errorf("unexpected statuses length, got %d, expected %d", len(resp.Data), len(pubKeys))
 	}
 
+	statuses = make([]web3signer.Status, 0, len(resp.Data))
 	for _, data := range resp.Data {
-		if data.Status != web3signer.StatusDeleted {
-			return fmt.Errorf("received status %s", data.Status)
-		}
+		statuses = append(statuses, data.Status)
 	}
 
-	return nil
+	return nil, nil
```

Comment:
This function should return the collected statuses, not nil.

Change line 180 from:
```go
return nil, nil
```
to:
```go
return statuses, nil
```
Also tests use blank identifier `_` to ignore the statuses return value, failing to validate the core functionality of this PR.